sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1079,7 +1079,7 @@ def _from_generated(cls, generated):
         if generated.properties.default_encryption_scope:
             scope = cls(
                 generated.properties.default_encryption_scope,
-                prevent_encryption_scope_override=generated.properties.deny_encryption_scope_override or False
+                prevent_encryption_scope_override=generated.properties.prevent_encryption_scope_override or False
```

Comment:
Does this header need updating as well? Or does that remain the same?
https://github.com/Azure/azure-sdk-for-python/blob/bea1bea93e2e432052432eedd07068f6fe66fd7c/sdk/storage/azure-storage-blob/azure/storage/blob/_models.py#L333

Reply:
Good question! The service header will not be updated